src/leap/mail/imap/mailbox.py

```diff
@@ -500,9 +500,9 @@ def fetch(self, messages_asked, uid):
         is_sequence = True if uid == 0 else False
 
         # XXX DEBUG ---  if you attempt to use the `getmail` utility under
-        # imap/tests, it will choke until we implement sequence numbers. This
-        # is an easy hack meanwhile.
-        # is_sequence = False
+        # imap/tests, or muas like mutt, it will choke until we implement
+        # sequence numbers. This is an easy hack meanwhile.
+        is_sequence = False
```

Comment:
Yes, I prefer to have it commented, but anyway I'll merge it as it is if you prefer it like that. The ugly hack is there anyway.